textdb/textdb-textql/src/test/java/edu/uci/ics/textdb/testql/statements/CreateViewStatementTest.java

```diff
@@ -0,0 +1,110 @@
+package edu.uci.ics.textdb.testql.statements;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import edu.uci.ics.textdb.textql.statements.CreateViewStatement;
+import edu.uci.ics.textdb.textql.statements.SelectExtractStatement;
+import edu.uci.ics.textdb.textql.statements.Statement;
+import edu.uci.ics.textdb.textql.statements.predicates.ExtractPredicate;
+import edu.uci.ics.textdb.textql.statements.predicates.KeywordExtractPredicate;
+import edu.uci.ics.textdb.textql.statements.predicates.SelectPredicate;
+import edu.uci.ics.textdb.textql.statements.predicates.SelectSomeFieldsPredicate;
+import edu.uci.ics.textdb.web.request.beans.OperatorBean;
+
+/**
+ * This class contain test cases for the CreateViewStatement. The constructor,
+ * getter, setter and bean builder methods are tested.
+ * 
+ * @author Flavio Bayer
+ *
+ */
+public class CreateViewStatementTest {
```

Comment:
A high-level comment: add high-level comments for each test case to explain its main idea.